public class TestException{
	public static void main(String[] args){
		int[] a = {1,2,3,4,5};
		try{
			System.out.println(a[5]);
		}
		catch(Exception e){
			System.out.println("The system is fixing, please wait a minute.");
			e.printStackTrace();
		}
		
		void m(int i){
			int b=10;
			int c = 0;
			
		}
	
	
	}
}

